package/MDAnalysis/converters/RDKitParser.py

```diff
@@ -90,16 +89,14 @@ class RDKitParser(TopologyReaderBase):
      - Atomnames
      - Aromaticities
      - Elements
+     - Types
```

Comment:
I'm not going to block over this, but these docstring changes are a bit confusing to me. At least the Atomtypes are still being guessed (even if it's done within tooling that isn't linked to the parser), so it's still makes sense to me that it be listed under "Guessed attributed" as before.

To explain my thoughts here - this list of attributes has historically been used as the "list of attributes which were read from file/object". By adding Types here we're removing that implicit documentation.

As a half-way solution, how about we at least annotate these a bit better, something like this?

```suggestion
     - Types (guessed through DefaultGuesser)
```